src/components/Activities/Activity1/index.tsx

```diff
@@ -92,6 +95,41 @@ function Activity1(): JSX.Element {
       topText: 'Computers have to learn just like we do: by trial and error!',
       child: <Slide3 />,
     },
+    {
+      child:
+        <div id="game2-intro">
+          <span>But even if we know what the alien is saying...can you figure out what they mean?</span>
+          <br/>
+          <span>Warning: One sentence can mean two things, so the answer might not be what you expect!</span>
+          <br/>
+          <button id="game2-intro-button" onClick={context.next}>
+            Play Game
+          </button>
+        </div>,
```

Comment:
something like this is probably what you want so that when `AmbiguousPhrasingGameIntro` is rendered, it is rendered within `Carousel`, so the `CarouselContext` will be passed properly!
```suggestion
     child: <MiniGame2Intro/>,
      showNext: false,
```